Cargo.toml

```
@@ -23,4 +23,4 @@ regex = "1.0"
 lazy_static = "1.1"
 derive-new = "0.5"
 petgraph = { version = ">=0.5,<0.7", optional = true }
-strum_macros = ">=0.20, <0.24"
+strum_macros = "0.24"
```

I see no reason not to keep this as a version range here, to remove the need for everyone to update everything in sync.  Nothing here is taking advantage of any features which are only present in the latest version.
```suggestion
strum_macros = ">=0.20, <0.25"
```